### PR TITLE
added setSeconds

### PR DESCRIPTION
**Pull request by ShivangRawat30**
### Description
added setSeconds.md at `docs/content/javascript/concepts/dates/terms/setSeconds/setSeconds.md`
solves #2500

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/javascript/concepts/dates/terms/setSeconds/setSeconds.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+Title: '.setSeconds()'
+Description: 'Set the seconds component of a date object to a specified value'
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setSeconds/setSeconds.md:3]:
```suggestion
Description: 'Sets the seconds component of a date object to a specified value.'
```

File: content/javascript/concepts/dates/terms/setSeconds/setSeconds.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  - 'Web Development'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'paths/web-development'
+  - 'introduction-to-javascript'
+---
+
+The **`.setSeconds()`** method takes an integer value as its parameter, representing the new value for the seconds.
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setSeconds/setSeconds.md:14]:
```suggestion
The **`.setSeconds()`** method sets the seconds component of a date object to a specified value. It takes an integer value as its parameter, representing the new value for the seconds.
```

File: content/javascript/concepts/dates/terms/setSeconds/setSeconds.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Syntax
+
+```pseudo
+myDate.setSeconds(secondsValue)
+```
+
+The returned value from `.setSeconds()` modifies the original date object by setting the seconds to the specified value.
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setSeconds/setSeconds.md:22]:
```suggestion
The returned value from `.setSeconds()` modifies the original date object, `myDate`, by setting the seconds to `secondsValue`.
```

File: content/javascript/concepts/dates/terms/setSeconds/setSeconds.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+## Example
+
+In the example below, the `myDate` variable takes the provided Time and then the `.setSeconds()` method is applied to return a modified time.
+
+```js
```
Comment (SSwiniarski) [inline content/javascript/concepts/dates/terms/setSeconds/setSeconds.md:28]: Can we use a static datetime value so the output will be consistent whenever we run this code?

**Review by SSwiniarski — COMMENTED** [review]
@ShivangRawat30 I did a first review and left a couple of comments. Tag me when you've addressed them.

**Review by SSwiniarski — APPROVED** [review]
@ShivangRawat30 Looks good for a second review.

File: content/javascript/concepts/dates/terms/setSeconds/setSeconds.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  - 'Web Development'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'paths/web-development'
+  - 'introduction-to-javascript'
+---
+
+The **`.setSeconds()`** method sets the seconds component of a date object to a specified value. It takes an integer value as its parameter, representing the new value for the seconds.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setSeconds/setSeconds.md:14]:
```suggestion
The **`.setSeconds()`** method sets the seconds component of a `Date` object to a specified value. It takes an integer value as its parameter, representing the new value for the seconds.
```

File: content/javascript/concepts/dates/terms/setSeconds/setSeconds.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+## Syntax
+
+```pseudo
+myDate.setSeconds(secondsValue)
+```
+
+The returned value from `.setSeconds()` modifies the original date object, `myDate`, by setting the seconds to `secondsValue`.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setSeconds/setSeconds.md:22]:
```suggestion
The returned value from `.setSeconds()` modifies the original `Date` object, `myDate`, by setting the seconds to `secondsValue`.
```

File: content/javascript/concepts/dates/terms/setSeconds/setSeconds.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+## Example
+
+In the example below, the `myDate` variable takes the provided Time and then the `.setSeconds()` method is applied to return a modified time.
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setSeconds/setSeconds.md:26]:
```suggestion
In the example below, the `myDate` variable is declared and then the `.setSeconds()` method is applied to return a modified time.
```

File: content/javascript/concepts/dates/terms/setSeconds/setSeconds.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+```js
+const myDate = new Date(August 29, 2022 23:15:30);
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setSeconds/setSeconds.md:29]:
```suggestion
const myDate = new Date('August 29, 2022 23:15:30');
```

File: content/javascript/concepts/dates/terms/setSeconds/setSeconds.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+myDate.setSeconds(42);
+
+console.log(myDate);
```
Comment (caupolicandiaz) [inline content/javascript/concepts/dates/terms/setSeconds/setSeconds.md:32]:
```suggestion
console.log(myDate.toString());
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@ShivangRawat30 I've added several comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by ShivangRawat30** [comment]
@caupolicandiaz I've implemented the suggested changes.

**Review by caupolicandiaz — APPROVED** [review]
@ShivangRawat30 Looks good

**Comment by SSwiniarski** [comment]
@ShivangRawat30 Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/javascript/dates/setSeconds

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**